templates/admin/organize/eventapplication/change_form.html

```diff
@@ -1,26 +1,32 @@
 {% extends "admin/change_form.html" %}
 {% load i18n %}
 
+{% block extrastyle %}
+{{ block.super }}
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
+{% endblock %}
+
```

Comment:
This page has different look from the rest of the admin pages. I am guessing it's missing some CSS style sheets. I have checked in both and Chrome and am getting the same result.

<img width="1679" alt="Screenshot 2021-09-07 at 18 44 52" src="https://user-images.githubusercontent.com/13593285/132383332-30a12723-9100-4bc2-a0f3-624c4d4ffde4.png">

Other pages look alright. See the screenshot below.
<img width="1678" alt="Screenshot 2021-09-07 at 18 44 23" src="https://user-images.githubusercontent.com/13593285/132383411-12d3ba7b-3188-47be-aeb9-9892c5afd693.png">

Reply:
That doesn't look like the stock admin 🤔

Reply:
You're right @carltongibson - that is django-suit.

@amakarudze admin shouldn't look like that anymore because this change removes django-suit. Admin should look like this now;

<img width="1341" alt="Screenshot 2021-09-07 at 18 41 11" src="https://user-images.githubusercontent.com/1461191/132387978-0184ecb1-81f9-4394-a934-8f10ae77814b.png">

Specifically here, it's the application admin which I think is the only one that has extra functionality. It provides the "Triaging" form which uses bootstrap classes so I've also included bootstrap CSS in this specific admin template - bootstrap used to be made available through django-suit.

Ultimately I wouldn't worry about the visuals of the admin for this PR because it's only django 2.2. Django 3 changes the admin with the introduction of the sidebar, so reviewing the visuals once we're at 3.2 would be best.

Speaking of which, I've revisited things for that screenshot and been able to address the lack of branding. I was leaving this for the 3.2 PR.

<img width="328" alt="Screenshot 2021-09-07 at 19 01 07" src="https://user-images.githubusercontent.com/1461191/132390196-0530c445-cbe3-4790-bcc3-81107310dbaa.png">

And on the subject of django 3.2 it's being held up by django-recaptcha. Tests are currently failing because it's not validating so I'm looking at options, including disabling recaptcha for tests.

Reply:
Ok, @marksweb, I needed to uninstall `django-suit` and `django-suit-redactor` to get the new interface. I guess I would need to uninstall them manually also in production.

I tried to migrate to Django 3.2 but couldn't get past the django-recaptcha glitch. Tests were failing so I just closed the PR.

Reply:
@amakarudze just run `pip-sync` after pulling or checking out to keep packages in sync.

Production should use this also to ensure old packages are removed.

Perhaps you can share the deploy script with me so I can advise?

And yeah, the django 3 move is more complicated than this one so it'll take me longer. I haven't raised a PR yet for it though, so not sure what you've closed.

Reply:
Oh @marksweb  I meant I had done my own PR for migrating to Django 3.2 which I closed after django-recaptcha messed up things lol. 

The deploy script only has `pip install -r requirements.txt` for managing packages. I just need to know if the command is no longer required if we introduce the commands you introduced with `pip-sync` or it's still required.